README.md

```diff
@@ -7,70 +7,35 @@ debt in the JS ecosystem.
 
-## The problem
+## How to help
 
-Packages often have dependencies, and those dependencies themselves have
-dependencies, and so on. This is fine and expected with how dependency trees
-are structured.
+All issues on our issue tracker are labeled and a lot of them contain actionable items you can start to work on right away.
+Here are some examples:
 
-However, over the years many of these deepest dependencies are long since
-obsolete utilities, polyfills, and so on. At the time they were introduced,
-they were important and had their use, but are now arguably redundant to
-most of us.
+<img src="https://img.shields.io/badge/needs%20first%20contact-b60205?style=flat" alt="needs first contact" style="vertical-align: middle"/>
 
-For example, polyfills for `Array.prototype.includes` are often found
-deep in most projects' dependency trees, and yet this feature has been available
-since Node 6 (released in ~2016).
+The maintainer of the repo was not contacted yet. Create an issue and ask per the repo's contributing guidelines.
 
-Similarly, we have all collected "micro packages" over time without realising
-it. These are very small (often one-liners) packages we depend on to do
-simple pieces of logic the JavaScript language already allows us to do. For
-example, `is-string`, `is-number`, `is-nan`, etc.
+If an issue was filed more than a couple months ago and hasn't received activity, you can also reach out to the maintainer via other channels listed on the repo but **make sure that contributions are welcome**. We want the maintainers on board and to convince them that this is important enough to spend time on.
 
-There are many reasons you could argue these are a bad idea. Security,
-complexity, install time, bundle size and so on. We unnecessarily download,
-install and include hundreds of these packages when the platform already
-provides the same functionality.
+<img src="https://img.shields.io/badge/accepts%20prs-DBE1EF?style=flat" alt="accepts prs" style="vertical-align: middle"/>
 
-## They have their uses
+The maintainer is welcoming PRs. Feel free to create PRs for all the issues you find. **Make sure to follow contribution guidelines!**
 
-Some of these packages do have their use, and a small group of consumers
-strongly disagree with removing them (usually the creators of said packages).
+<img src="https://img.shields.io/badge/needs%20alternative-F32096?style=flat" alt="needs alternative" style="vertical-align: middle"/>
 
-For example, one primary argument for using the polyfill-like packages is
-that a third-party package can override built-ins and break another package's
-code. To counter this, these packages use a custom implementation/wrapper around
-the native functionality instead of using the native functionality directly
-(since nobody can override the wrapper).
+The package in question needs a replacement that we can bring forward. Feel free to create one, but beware: its maintenance burden is on you!
```

Comment:
I'll bring this one up in discord but the tldr is I think we should say something like: 

> You should look for a replacement package and, if an appropriate one doesn't exist, consider making a new one. Though keep in mind new packages should be well tested and the community made aware of them before any migrations